docs/README.md

```diff
@@ -0,0 +1,171 @@
+# Shortest
+
+[![npm version](https://badge.fury.io/js/shortest.svg)](https://badge.fury.io/js/shortest)
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](http://makeapullrequest.com)
+
+> 🤖 An AI-powered E2E testing framework that uses natural language to write and execute tests.
+
+## ✨ Features
+
+- 🤖 AI-powered test execution
```

Comment:
Seems AI written and not useful. Better to be on the homepage than README if we think it's needed